### PR TITLE
Fix small issues, mostly docs (fixes #434)

### LINKED ISSUE #434: Documentation issues and a mistake in code
Dear DeerLab team,

please note some corrections to the documentation proposed by me. In addition, there is a small mistake found in `dd_models.py`. Everything is described in the attached file.

Thank you

[DeerLab documentation issues.docx](https://github.com/JeschkeLab/DeerLab/files/12260726/DeerLab.documentation.issues.docx)

### PR DESCRIPTION
**Pull request by stestoll**

### CODE REVIEW

File: docsrc/source/dipolar_guide_modeling.rst [hunk 1]
```diff
@@ -3,41 +3,55 @@
...
+* **Background model** ``Bmodel``: Describes the dipolar background signal arising from the inter-molecular spin pairs. 
+
+* **Type of experiment** ``experiment``: This sets the number of dipolar pathways contributing to the dipolar signal.
+
+For each of these components, a choice needs to be made: 
+
+Choosing a time range
+*************************
+
+The time range depends on the expected range of distances, but is typically in the range of 0.5 to several microseconds. To construct a time vector, use the ``linspace`` function from NumPy: ::
```
Comment (HKaras) [inline docsrc/source/dipolar_guide_modeling.rst:27]: I am unsure of this wording here. When creating the model you have already got your data, so you aren't so much choosing your time range more inputting it. I also wouldn't say that the time range is dependent on the distances, more the range of extractable distances is dependent upon the time axis. 

Additionally most people use a tau1 of 0.4us for 4pDEER, so the range starts before 0.5 us.

**Review by HKaras — APPROVED** [review]
It all looks good except I don't fully agree with the language around 'Choosing a time range'.

**Comment by stestoll** [comment]
I modified the language. Good now?

**Review by HKaras — APPROVED** [review]
Looks good thanks.

**Pull request merged**